homeassistant/loader.py

```diff
@@ -875,6 +875,21 @@ def _resolve_integrations_from_root(
     return integrations
 
 
+def async_get_loaded_integration(hass: HomeAssistant, domain: str) -> Integration:
```

Comment:
We can't decorate this with @callback because it results in a circular dependency between core and loader.

An option would be to only import `loader` locally in core though.

Reply:
resolved via 974c014d00848e0cf866fa409d3d4ccc0c8ea136